features/borrowers_view_and_sign_deed.feature

```diff
@@ -0,0 +1,280 @@
+@borrowers_view_and_sign_deed
+
+Feature: Borrowers can view and sign a deed
+As a borrower/borowers I need to confirm that the details of my mortgage deed are correct
+So that the mortgage can proceed to the next stage.
+
+Acceptance Criteria:
+(US133) Each borrower must be able to confirm the details are correct.
+(US133) By confirming the details are correct, the deed is digitally signed.
+(US133) Confirmation Message displayed to borrower on a subsequent page.
+(US133) Each borrower must be able to view a deed multiple times but must only be able to confirm the details once.
+(US133) When the deed is viewed subsequently it will show that the relevant borrower has confirmed the deed.
+
+@us133
+Scenario: Single borrower views and signs deed, then views again
+  Given I have valid deed data with <1> borrowers
+  And I create the deed via the Deed API
+  And borrower <1> views the deed
+  And the borrower signature element <1> is present on page
+  Then the deed is digitally signed by borrower <1>
+  And borrower <1> views the deed again
+  Then I verify borrower <1> has signed the deed
+
+@us133
+Scenario: Single borrower views deed, then views and signs
+  Given I have valid deed data with <1> borrowers
+  And I create the deed via the Deed API
+  And borrower <1> views the deed
+  And borrower <1> views the deed again
+  Then the deed is digitally signed by borrower <1>
+
+@us133
+Scenario: Single borrower views deed 3 times
+  Given I have valid deed data with <1> borrowers
+  And I create the deed via the Deed API
+  And borrower <1> views the deed
+  And borrower <1> views the deed again
+  Then borrower <1> views the deed again
```

Comment:
As discussed with @nigelpain this is an important area, and high test coverage was decided as a good approach